src/fusor/examples/bcr_abl1.json

```diff
@@ -5,29 +5,18 @@
       "type": "TranscriptSegmentElement",
       "transcript": "refseq:NM_004327.3",
       "gene_descriptor": {
```

Comment:
we should also change this key name to `gene`

Reply:
Agreed, I will update this naming